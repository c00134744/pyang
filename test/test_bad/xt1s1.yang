submodule xt1s1 {
  belongs-to xt1;
  import xt3 { prefix xt3p; } // different prefix than in xt1
  organization "hopp";
  contact "hej" {
    xt3p:test hej {}
    xt3:test hej {xt3:hemm he;} // error: bad prefix
  }
  revision 2007-06-06 { description "initinla"; }
}
  

