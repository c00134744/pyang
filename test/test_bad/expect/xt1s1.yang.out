xt1s1.yang:7: error: PREFIX_NOT_DEFINED
xt3.yang:4: error: UNEXPECTED_KEYWORD_1
